openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/registry.py

```diff
@@ -44,6 +41,5 @@
 build_registry.add(PipelineType.dbt.value)(build_dbt_dag)
 build_registry.add(PipelineType.profiler.value)(build_profiler_dag)
 build_registry.add(PipelineType.TestSuite.value)(build_test_suite_dag)
-build_registry.add(PipelineType.dataInsight.value)(build_data_insight_dag)
```

Comment:
On a separate PR, but remember to pop a docs change and highlight we are removing the python side of the DI workflow

Reply:
Absolutely right. I'll add it to the TODO to be done before the release is done